src/front/zokrates/mod.rs

```diff
@@ -611,6 +585,8 @@ impl<'ast> ZGen<'ast> {
         for (p, f) in &t {
             self.file_stack.push(p.to_owned());
             for func in &f.functions {
+                println!("fn {} in {}", func.id.value, self.cur_path().display());
+
```

Comment:
Remove this? Also, see [here](https://docs.rs/env_logger/latest/env_logger/index.html) for how to use the logging infrastructure. Quickstart: set `RUST_LOG=debug ` before running the binary.